ocaml/xapi/xapi_vm.ml

```diff
@@ -551,6 +553,15 @@ let create ~__context ~name_label ~name_description
     ~nVRAM
   : API.ref_VM =
 
+  (* `ignore` is necessary here or the compilation will break because this method doesn't return in this error cases *)
```

Comment:
Small simplifications:
* ` ignore @@ match ...`
*  `raise Api_errors.(Server_error (vm_bad_power_state, …`
* please avoid overlong lines - they are hard to review

Reply:
Done! I split the `raise` lines to be more readable. :)